src/LiveQuery/RequestSchema.js

```diff
@@ -70,6 +70,22 @@ const subscribe = {
           minItems: 1,
           uniqueItems: true,
         },
+        keys: {
```

Comment:
@dblythy Does `keys` and `watch` need to be added to `update` like `fields` on line 119 below?

Reply:
Nice pickup!!